File: java/json-transform/src/main/java/co/nlighten/jsontransform/functions/TransformerFunctionRaw.java
package co.nlighten.jsontransform.functions;

import co.nlighten.jsontransform.functions.common.FunctionContext;
import co.nlighten.jsontransform.functions.common.TransformerFunction;

public class TransformerFunctionRaw extends TransformerFunction {
    public TransformerFunctionRaw() {
        super();
    }
    @Override
    public Object apply(FunctionContext context) {
        return context.getJsonElement(null, false);
    }
}

File: javascript/json-transform/src/functions/TransformerFunctionRaw.ts
import TransformerFunction from "./common/TransformerFunction";
import FunctionContext from "./common/FunctionContext";

class TransformerFunctionRaw extends TransformerFunction {
  constructor() {
    super({});
  }

  override async apply(context: FunctionContext): Promise<any> {
    return context.getJsonElement(null, false);
  }
}

export default TransformerFunctionRaw;
